lighthouse-core/test/gather/gatherers/js-usage-test.js

```diff
@@ -45,10 +46,20 @@ describe('JsUsage gatherer', () => {
     const driver = new Driver(connectionStub);
```

Comment:
somewhat up to you when to tackle this, but we'll eventually need to migrate gathering tests to the new mock driver/sessions in https://github.com/GoogleChrome/lighthouse/blob/0b00b5c667bf40b5945632245bf8e803a9843b40/lighthouse-core/test/fraggle-rock/gather/mock-driver.js like the driver/gather-runner PRs have been doing 

I think I agree with the argument for leaving these as-is for now and making a second pass when FR becomes the default, but wanted it on your radar so you're not surprised when the time eventually comes for it :)